hms-app/src/components/input-forms/IdeaForm.tsx

```diff
@@ -52,6 +52,7 @@ function IdeaForm(props: IProps) {
     goal: '',
     creationDate: new Date(),
   })
+  const allowedIdeaTitleLength = 100
```

Comment:
it would be best to define this in a single place so that it is hard to make the mistake of having two values.  Maybe somewhere close to the Idea interface since it is tied to that.

Reply:
also maybe 128 :P nice power of 2